_dashboards/discover/multi-data-sources.md

```diff
@@ -0,0 +1,131 @@
+
+## Create a data source connection
+
+A data source connection specifies the parameters needed to connect to a data source. These parameters form a connection string for the data source. In Dashboards, you can add new data source connections or edit existing connections. 
+
+To create a new data source connection:
+
+1. Open Dashboards. If you’re not running the security plugin, go to [`http://localhost:5601`](http://localhost:5601/). If you’re running the security plugin, go to [`https://localhost:5601`](https://localhost:5601/) and log in with the username `admin` and password `admin`.
+
+2. In the Dashboards console, select **Stack Management > Data Sources `Experimental` > Data Sources > Create data source connection**.
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-1.png" width=500 alt="Data sources user interface">
+
+3. Add information to each field to configure **Connection Details**, **Endpoint**, and **Authentication** to connect to a data source. For this tutorial, the **Endpoint URL** is `http://localhost:5601/app/management/opensearch-dashboards/dataSources`.
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-2.png" width=500 alt="Create a data source connection user interface">
+
+4. Select  **Create data source connection** to save your settings.
+
+5. Return to the **Data Sources** main page to confirm the newly created data source is listed under **Data Sources**.
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-3.png" width=500 alt="Data sources list user interface">
+
+6. (Optional): Select the data source to verify settings are configured properly.
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-4.png" width=500 alt="Data sources settings verification user interface">
+
+## Create  an index pattern
+
+Index patterns allow you to access the OpenSearch data that you want to explore. An index pattern selects the data to use and allows you to define the field properties. Learn how to load your own data and create an index pattern following these steps. This tutorial uses the preconfigured index pattern `opensearch_dashboards_sample_data_ecommerce Default`.
+
+1. In the Dashboards console, select **Index Patterns > Create index pattern**.
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-5.png" width=500 alt="Index pattern user interface">
+
+2. Select **Use external data source connection**.
+3. Start typing in the Search data sources field to search for the data source you created earlier and then select the data source and **Next step**.
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-6.png" width=500 alt="Index pattern search user interface">
+
+4. Add an **Index pattern name** to define the index pattern and then select **Next step**.
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-7.png" width=500 alt="Index pattern define user interface">
```

Comment:
This screen shot is not accurate, due to cannot use input `sample` to create index pattern. You can try use `opensearch*` instead

Reply:
replaced image